submissions/Roophee/JS - DOM/index.js

```diff
@@ -0,0 +1,111 @@
+"use strict";
+
+const clickBurger = document.querySelector(".burger");
+const clickContent = document.querySelector(".content");
+const clickHeader = document.querySelector(".header");
+const clickfooter = document.querySelector(".footer");
+
+const deBar = document.querySelector(".aside");
+
+
+// MOVE ASIDE & CONTENT
+
+function hideSideBar(e) { 
+    e.preventDefault();
+    deBar.classList.toggle('hide__sidebar');
+    clickContent.classList.toggle('move_content');
+    clickBurger.classList.toggle('burger__stop');
+    console.log(e.type)
```

Comment:
Avoid committing `console.log` )

Reply:
Fixed all remarks. Thanks.
Please, check it.

Reply:
All done. Check it. Thanks.